src/lint/linter/ArcanistGoFmtLinter.php

```diff
@@ -27,9 +27,17 @@ public function getLinterConfigurationName() {
   }
 
   public function getBinary() {
+    $go_imports = $this->getGoImportsBinary();
+    if (Filesystem::binaryExists($go_imports)) {
+      return $go_imports;
+    }
     return 'gofmt';
   }
 
+  public function getGoImportsBinary() {
+    return 'goimports';
```

Comment:
It seems a little cleaner than hardcoding it all in `getBinary()`, this way another message (e.g. an error message) could tell you how to install `goimports`